src/Models/ModelInstantiator.php

```diff
@@ -325,4 +329,28 @@ public function convertPropertyOnRetrieve(
 
         return $column->getConverter()->onRetrieve($property);
     }
+
+    /**
+     * @param mixed $property
+     * @param array $property_reflection
+     * @return mixed
+     */
+    private function getReflectionProperty(mixed $property, array $property_reflection): mixed
```

Comment:
Looks like the return type may always be an array.

Reply:
> Looks like the return type may always be an array.

This array coming in is an array of ReflectionProperty objects:

```
Array
(
    [0] => ReflectionProperty Object
        (
            [name] => pdgaNumber
            [class] => PDGA\DataObjects\Models\Test\ModelInstantiatorTestObject
        )

    [1] => ReflectionProperty Object
        (
            [name] => firstName
            [class] => PDGA\DataObjects\Models\Test\ModelInstantiatorTestObject
        )
```

It's a little confusing because the `array_column` function can take an array of arrays or array of objects. In this case it's objects and it's pulling out the `name` property and searching it.

In this case, I changed `mixed` to `ReflectionProperty` and the tests pass still.